app/src/main/java/org/projectbuendia/client/ui/dialogs/EditPatientDialogFragment.java

```diff
@@ -98,16 +87,16 @@ private void populateFields(Bundle args) {
         LocalDate birthdate = Utils.toLocalDate(args.getString("birthdate"));
         if (birthdate != null) {
             Period age = new Period(birthdate, LocalDate.now());
-            mAgeYears.setText("" + age.getYears());
-            mAgeMonths.setText("" + age.getMonths());
+            mAgeYears.setText(String.valueOf(age.getYears()));
+            mAgeMonths.setText(String.valueOf(age.getMonths()));
         }
-        Integer gender = args.getInt("gender");
-        if (gender != null) {
-            if (gender == Patient.GENDER_FEMALE) {
+        switch (args.getInt("gender", Patient.GENDER_UNKNOWN)) {
```

Comment:
The idea here was:
- `args.getInt("gender")` doesn't return an `Integer`, it returns an `int`, and returns the value `0` if the bundle didn't contain the specified key.
- The old code only worked incidentally because `Patient.GENDER_UNKNOWN == 0`. 
- I changed it to a `switch` instead of an `if` because I thought it was more readable - `switch` implies that you're handling all possible cases, whereas `if` / `else if` is more readable for isolated checks. i.e. if we were just checking against one of the three possible values, I would have left it as an if, but seeing as we're checking against all possible values, it made more sense as a switch.